ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/CveStatusTabNavigation.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import {
+    Tabs,
+    Tab,
+    TabTitleText,
+    TabsComponent,
+    PageSection,
+    Card,
+    CardBody,
+} from '@patternfly/react-core';
+
+import { vulnerabilitiesWorkloadCvesPath } from 'routePaths';
+import { getQueryString } from 'utils/queryStringUtils';
+import WorkloadTableToolbar from './WorkloadTableToolbar';
+import EntityTypeToggleGroup from './EntityTypeToggleGroup';
+import { WorkloadCvesSearch } from './searchUtils';
+
+const observedCvesQueryString = getQueryString<WorkloadCvesSearch>({ cveStatusTab: 'Observed' });
+const observedCvesPath = `${vulnerabilitiesWorkloadCvesPath}${observedCvesQueryString}`;
+
+function CveStatusTabNavigation() {
+    const [activeTabKey, setActiveTabKey] = useState(0);
+
+    function handleTabClick(e, tabIndex) {
+        setActiveTabKey(tabIndex);
+    }
```

Comment:
nit: I noticed a weird little effect. When I click on the only active tab, it causes a real browser refresh of the page.